meteor/jest.config.js

```diff
@@ -19,7 +19,7 @@ const commonConfig = {
 			},
 		},
 	},
-	moduleFileExtensions: ['ts', 'js'],
+	moduleFileExtensions: ['ts', 'js', 'tsx', 'jsx'],
```

Comment:
Do we need to include TSX and JSX in `moduleFileExtensions`? We currently have no way of unit testing React code, and all JSX we use is React.

Reply:
Good point! I added that while trying to get Jest to run a tsx-file, but I ended up splitting the file into a pure ts-file instead, and mistakenly left it there.
I'll revert it, since it's not used anywhere.